eng/targets/XUnit.targets

```diff
@@ -11,16 +11,20 @@
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)..\config\test\BannedSymbols.txt" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'">
+    <Compile Include="$(RepositoryEngineeringDir)config\test\Core\InstallTraceListener$(DefaultLanguageSourceExtension)"
+             Condition="Exists('$(RepositoryEngineeringDir)config\test\Core\InstallTraceListener$(DefaultLanguageSourceExtension)')"/>
+  </ItemGroup>
```

Comment:
Module initializers work on .NET Framework too so why aren't we just universally doing this?

Reply:
With this pull request, the following are covered:

* .NET Framework test projects (via app.config)
* .NET Core test projects written in C# (via module initializer)
* .NET Core test projects written in Visual Basic that use `<UseExportProvider>`

Keeping app.config for .NET Framework prevents a test gap for some Visual Basic projects.

Reply:
Hmm, I think another way of stating is that there is a gap in VB for .NET Core projects with this change. Essentially paths that don't go through `<UseExportProvider>`. This gap does not exist on desktop because of the `app.config` work. Correct?

Reply:
Correct. Xunit doesn't offer an AssemblyInitialize extension point, and Visual Basic does not offer module initializers, so we do not have a way to guarantee that the trace listener is installed by the time the first test executes.

Reply:
K. Think we should doc that so it's clearer what this still gives us.

Reply:
Will submit a follow-up PR to document the condition here.